RGBMatrixEmulator/emulators/options.py

```diff
@@ -59,6 +59,8 @@ def __init__(self):
             Logger.warning('"{}" pixel style option not recognized. Valid options are "square", "circle". Defaulting to "square"...'.format(config_pixel_style))
 
         self.pixel_size = emulator_config.pixel_size
+        self.pixel_outline = emulator_config.DEFAULT_CONFIG.get('pixel_outline')
+        self.pixel_outline = emulator_config.pixel_outline
```

Comment:
The default here is redundant, the options class merges defaults into the user-defined config on initialization

Reply:
Good point, I've fixed it!